### PR TITLE
4th homework

### PR DESCRIPTION
**Pull request by ryvik13**

### CODE REVIEW

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  align-items: center;
+  height: 160px;
+}
+
+.footer-container {
+  width: 100%;
+}
+
+.footer-box-content {
+  width: 90%;
```
Comment (anna-vrana-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:179]: If you want to limit the width, use `margin: 0 auto;` to center the content

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  height: 42px;
+  width: 192px;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 33px;
+  position: relative;
+  list-style-type: none;
```
Comment (anna-vrana-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:196]: It's better to use the `position: relative` property on the parent container. Please remove it from here

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+}
+
+.icon-fb {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
```
Comment (anna-vrana-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:203]: Add `position: relative` here

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+  top: 11px;
+  position: absolute;
+}
+
+.icon-link {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
```
Comment (anna-vrana-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:220]: Add `position: relative` here

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+  left: 86px;
+  top: 11px;
+  position: absolute;
+}
+
+.icon-inst {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
```
Comment (anna-vrana-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:235]: Add `position: relative` here

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+}
+
+.copyright {
+  color: #EEF1EF;
+  margin-bottom: 0;
+  font-weight: 400;
+  line-height: 32px;
+  font-size: 22px;
+  text-align: center;
+}
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:254]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+.login-form-checkbox::before {
+  content: "";
+  color: transparent;
+  display: block;
+  width: inherit;
+  height: inherit;
+  border-color: #FFF;
+  background-color: transparent;
+  background-size: contain;
+}
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:154]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/login.html [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+        </header>
+        <main>
+            <div class="login-form">
+                <form action="success.html">
+                    <h3 class="login-form-title">Login page</h3>
+                    <div class="login-form-group">
+                        <label class="login-form-label" for="email">Email</label>
+                        <input class="login-form-input" type="email" name="email" id="email" placeholder="Enter your email address" pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
+                    </div>
+                    <div class="login-form-group">
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/login.html:44]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 1 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:179]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 3 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:203]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 4 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:220]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 5 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:235]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 6 above)
[the remark this answers is not in the document]
Comment (anna-vrana-moc) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:257]: You should import this font on the page before using it. You can do so inside the `<head>` tag using a link or by `@import` directly into the CSS file

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/login.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (anna-vrana-moc) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/login.html:44]: It's an unnecessary wrap `<div class="login-form-group">;` you can style the label and input. Please remove it

**Comment by anna-vrana-moc** [comment]
hey @ryvik13, there are still 5 unresolved issues. Please review your code to fix these remaining problems

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 5 above)
Comment (anna-vrana-moc) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:237]: Please adjust the positions. Aligning icons to the list is not a good idea. Instead, fix the positioning inside the list items

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css (hunk 3 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:205]: I added but all icons moved to high lift corner

Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/styles/login.css:205]: I haven't idea how to fix it

**Comment by ryvik13** [comment]
fixed all review comments

**Pull request merged**